templates/serverpod_templates/github/workflows/deployment-docker.yml

```
@@ -0,0 +1,157 @@
+        run: |
+          install -m 600 -D /dev/null ~/.ssh/id_rsa
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+          ssh-keyscan -H ${{ secrets.SSH_HOST }} >> ~/.ssh/known_hosts
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: List files in the directory
+        run: ls -la ./projectname_server/
+
+      - name: Run docker-compose
+        run: |
+          docker compose -f ./projectname_server/docker-compose.production.yaml pull
+          docker compose -f ./projectname_server/docker-compose.production.yaml up -d
+        env:
+          DOCKER_HOST: "ssh://${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }}"
+          # The database name
+          POSTGRES_DB: ${{ secrets.SERVERPOD_DATABASE_NAME }}
+          # The user name for the database
+          POSTGRES_USER: ${{ secrets.SERVERPOD_DATABASE_USER }}
+          # The password for the database
+          POSTGRES_PASSWORD: ${{ secrets.SERVERPOD_DATABASE_PASSWORD }}
+          # The port number for the API server
+          SERVERPOD_API_SERVER_PORT: ${{ secrets.SERVERPOD_API_SERVER_PORT }}
+          # The public host address of the API server
+          SERVERPOD_API_SERVER_PUBLIC_HOST: ${{ secrets.SERVERPOD_API_SERVER_PUBLIC_HOST }}
+          # The public port number for the API server
+          SERVERPOD_API_SERVER_PUBLIC_PORT: ${{ secrets.SERVERPOD_API_SERVER_PUBLIC_PORT }}
+          # The public scheme (http/https) for the API server
+          SERVERPOD_API_SERVER_PUBLIC_SCHEME: ${{ secrets.SERVERPOD_API_SERVER_PUBLIC_SCHEME }}
+          # The port number for the Insights server
+          SERVERPOD_INSIGHTS_SERVER_PORT: ${{ secrets.SERVERPOD_INSIGHTS_SERVER_PORT }}
+          # The public host address of the Insights server
+          SERVERPOD_INSIGHTS_SERVER_PUBLIC_HOST: ${{ secrets.SERVERPOD_INSIGHTS_SERVER_PUBLIC_HOST }}
+          # The public port number for the Insights server
+          SERVERPOD_INSIGHTS_SERVER_PUBLIC_PORT: ${{ secrets.SERVERPOD_INSIGHTS_SERVER_PUBLIC_PORT }}
+          # The public scheme (http/https) for the Insights server
+          SERVERPOD_INSIGHTS_SERVER_PUBLIC_SCHEME: ${{ secrets.SERVERPOD_INSIGHTS_SERVER_PUBLIC_SCHEME }}
+          # TODO hardcode  The port number for the Web server
+          SERVERPOD_WEB_SERVER_PORT: ${{ secrets.SERVERPOD_WEB_SERVER_PORT }}
+          # The public host address of the Web server
+          SERVERPOD_WEB_SERVER_PUBLIC_HOST: ${{ secrets.SERVERPOD_WEB_SERVER_PUBLIC_HOST }}
+          # The public port number for the Web server
+          SERVERPOD_WEB_SERVER_PUBLIC_PORT: ${{ secrets.SERVERPOD_WEB_SERVER_PUBLIC_PORT }}
+          # The public scheme (http/https) for the Web server
+          SERVERPOD_WEB_SERVER_PUBLIC_SCHEME: ${{ secrets.SERVERPOD_WEB_SERVER_PUBLIC_SCHEME }}
```

```suggestion
          SERVERPOD_WEB_SERVER_PUBLIC_SCHEME: "https"
```
should default to https since we do all of the traefik stuff for exactly this